internal/cli/kraft/pkg/packager_kraftfile_runtime.go

```diff
@@ -292,9 +292,14 @@ func (p *packagerKraftfileRuntime) Pack(ctx context.Context, opts *PkgOptions, a
 		}
 	}
 
-	cmdShellArgs, err := shellwords.Parse(strings.Join(opts.Args, " "))
-	if err != nil {
-		return nil, err
+	args = []string{}
+
+	// Only parse arguments if they have been provided.
+	if len(opts.Args) > 0 {
+		args, err = shellwords.Parse(fmt.Sprintf("'%s'", strings.Join(opts.Args, "' '")))
```

Comment:
I see the Parse changed, I hope it's ok, I'll try it out.

Reply:
I get this now, it's so counterintuitive, cause you set the `'` in between words, and then the start/end :D

I guess it will work